documentation/universal_links.md

```diff
@@ -0,0 +1,250 @@
+# Universal Links
+
+- [Universal Links](#universal-links)
+  - [What are Universal Links?](#what-are-universal-links)
+  - [Setting Up an Entitlements File for Universal Links](#setting-up-an-entitlements-file-for-universal-links)
+    - [Steps to Configure the Entitlements File](#steps-to-configure-the-entitlements-file)
+    - [Points to Consider](#points-to-consider)
+  - [Configuring Universal Links for your Storefront](#configuring-universal-links-for-your-storefront)
+  - [Important: The catch-all route `*`](#important-the-catch-all-route-)
+  - [Handling Universal Links in your app](#handling-universal-links-in-your-app)
+  - [Security Considerations](#security-considerations)
+  - [Further Reading and Resources](#further-reading-and-resources)
+
+## What are Universal Links?
+
+Universal Links are an iOS feature which allow apps to open links directly in an app rather than sending the user to a browser. This feature provides a seamless user experience by directly engaging app users with relevant content.
```

Comment:
This content is mostly regarding Universal Links. Consumers won't need to set up deep links for universal links to work

Reply:
Fair - I only thought a high-level because the description of universal links here applies 1:1 to deep links, wasn't sure whether confusion may arise or a reassurance that this is a more secure mechanism